plugins/kubernetes-backend/api-report.md

```diff
@@ -191,9 +199,16 @@ export interface KubernetesFetcher {
 //
 // @public (undocumented)
 export interface KubernetesObjectsProvider {
+  // (undocumented)
+  getCustomResourcesByEntity(
+    entity: Entity,
+    auth: AuthConfig,
+    customResources: CustomResourceMatcher[],
```

Comment:
Could of parameters being lined up here, perhaps worth moving over to an options object instead? `({entity, auth, customResources})`, or something of that sort, could also be `(entity, { auth, customResources })` e.g.

Reply:
`entity` will be removed in the next PR when the old endpoint is removed, which might shake things up a bit